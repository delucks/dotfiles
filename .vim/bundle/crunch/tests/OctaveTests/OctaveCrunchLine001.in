" Multiplication Test
sin(1)sin(1)
2sin(1)
sin(1)2
(2*3)(3*2)
(2*3)(3*2)
2(3*2)

" Integer Conversion Test
1/2

" Decimal w/o Leading Zeros Conversion
.5/2
" Remove Zeros In Result
.25*4
2^8
