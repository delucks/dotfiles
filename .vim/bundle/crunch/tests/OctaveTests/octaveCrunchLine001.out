" Multiplication Test
sin(1)sin(1) = 0.70807
2sin(1) = 1.6829
sin(1)2 = 1.6829
(2*3)(3*2) = 36
(2*3)(3*2) = 36
2(3*2) = 12

" Integer Conversion Test
1/2 = 0.5

" Decimal w/o Leading Zeros Conversion
.5/2 = 0.25
" Remove Zeros In Result
.25*4 = 1
2^8 = 256
